RTL_testcases/RTL_Benchmarks_Gap_Analysis/sdio_client_top/raptor_sdc.sdc

```diff
@@ -1,3 +1,3 @@
-create_clock -period 2.5 sdclk
+create_clock -period 2.5 -name sdclk
```

Comment:
Same here. This is incorrect, with the -name option you are creating a virtual clock. sdclk is a real clock.
It is not handled properly, please file a Jira.
Do not create virtual clocks in general.